TestResultSummaryService/parsers/BenchmarkMetric.js

```diff
@@ -235,7 +235,106 @@ const BenchmarkMetricRegex = {
                 funcName: geomean,
             }
         }
-    }
+    },
+    quarkus_startup: {
+        outerRegex: /Quarkus Test:  measure \d*? ([\s\S\n]*)/,
+        //We print Quarkus Test: measure n after running warmup runs
+        //Example: Quarkus Test:  measure 3 ~ measure run text
+        metrics: {
+            "Footprint": {
+                //Example: Footprint for  warmup 0 10129332
+                regex:/Footprint for  measure \d*? (\d*\.?\d*)/,
+                higherbetter: false,
+                units: "kb",
+                },
+            "Startup time in ms": {
+                //Example: Startup time for measure 0 : 53232
+                regex:/Startup time for measure \d*? : (\d*\.?\d*)/,
+                higherbetter: false,
+                units: "ms",   
+            }
+        }
+    },
+    pingperf_throughput: {
+        outerRegex:/Running \d*? measures([\s\S\n]*)/, 
+        //We print Running n measures after running warmup runs
+        //Example: Running 3 measures ~ measure run text
+        metrics: {
+            "Baremetal throughput": {
+                //Example: Requests/sec: 100522.45/
+                regex:/Requests\/sec: (\d*\.?\d*)/,
+                higherbetter: true,
+                units: "Requests/sec"
+            },
+        }
+    },
+    restCrud_quarkus: {
+        outerRegex:/Measure \d*?([\s\S\n]*)/,
+        //We print Running n measures after running warmup runs
+        //Example: Running 3 measures ~ measure run text
+        metrics: {
+            "Throughput": {
+                //Example: Requests/sec: 15145.27
+                regex:/Requests\/sec: (\d*\.?\d*)/,
+                higherbetter: true,
+                units: "Requests/sec",
+            },
+        }
+    },
+    restCrud_multi_user_quarkus: {
+        metrics: {
+            "Throughput": {
+                //Example: Requests/sec: 15145.27  -- in multi user, we are running multiple scenarios in 1 run - using geomean as our metric.
+                regex:/Measure[\s\S\n]*?Requests\/sec: (\d*\.?\d*)/,
+                higherbetter: true,
+                units: "Requests/sec",
+                funcName: geomean,
+            },
+        }
+    },
```

Comment:
- regex in `restCrud_multi_user_quarkus ` does not match its example
- can `restCrud_quarkus ` and `restCrud_multi_user_quarkus ` use the same parse logic?

Reply:
For restCrud_quarkus - we just run 1 scenario of 40 threads & 40 clients.
For multi_user test, though they share same output format, we are running multiple thread & connection scenarios(5,10,20,40,..) in 1 run so we thought it'd be better if we use geomean when we run it.
If seemed little odd for us to use geomean for 1 throughput result(non multi user run) so we separated it to 2 different logics but let me know if you think there should be only 1 parser logic for both.

Reply:
To avoid redundancy, maybe we can just use geomean for both.